vnext/Microsoft.ReactNative/ReactNativeHost.cpp

```diff
@@ -185,4 +189,15 @@ void ReactNativeHost::OnBackPressed() noexcept {
   //}
 }
 
+void ReactNativeHost::AddLifecycleEventListener(ILifecycleEventListener const& listener) {
```

Comment:
>ReactNativeHost [](start = 5, length = 15)

ReacNativeHost is not exposed to native modules and never must be to avoid circular references. I wonder what is planned to subscribe to these events.